### PR TITLE
Fixes #5564 : MultiTenancy UrlHelperExtensions throwing if RequestUrl…

### LINKED ISSUE #5564: [dev] MultiTenancy UrlHelperExtensions throwing if RequestUrlHost null
*dev branch only*
Exception is raised on after creating a new tenant without setting RequestUrlHost.

### PR DESCRIPTION
**Pull request by TFleury**
…Host null

### CODE REVIEW

File: src/Orchard.Web/Modules/Orchard.MultiTenancy/Extensions/UrlHelperExtensions.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@ namespace Orchard.MultiTenancy.Extensions {
     public static class UrlHelperExtensions {
         public static string Tenant(this UrlHelper urlHelper, ShellSettings tenantShellSettings) {
 
-            var requestUrlHost = tenantShellSettings.RequestUrlHost.Split(new[] { "," }, StringSplitOptions.RemoveEmptyEntries).First();
+            var requestUrlHost = tenantShellSettings.RequestUrlHost?.Split(new[] { "," }, StringSplitOptions.RemoveEmptyEntries).First();
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.MultiTenancy/Extensions/UrlHelperExtensions.cs]: Can't use C# 6

**Comment by TFleury** [comment]
OK, I will push a new PR.

**Comment by DanielStolt** [comment]
Don't bother with a new PR @TFleury I'll fix this in dev.

**Comment by TFleury** [comment]
Already pushed #5566

**Pull request closed without merging**